app/main.py

```diff
@@ -1 +1,35 @@
-# write your code here
+import os
+from app.errors import check_for_errors, WrongCommand, NoSourceFile
+
+
+def move_file(command: str) -> None:
+    try:
+        check_for_errors(command)
+
+    except (WrongCommand, NoSourceFile) as e:
+        print(e)
+        return e
+
+    command_name, filename, move_filename = command.split()
+
+    current_dir = os.getcwd()
```

Comment:
you can smplify this code 

    command_name, source_name, path_new_file = command.split()
    path, filename = os.path.split(path_new_file)

    if path:
        os.makedirs(path, exist_ok=True)

Reply:
and after this use `with`